torch/_refs/__init__.py

```diff
@@ -3327,7 +3329,7 @@ def roll(
         assert len_dims > 1
         tail_shifts = shifts[1:]
         tail_dims = dims[1:]
-        first_dim_rolled = torch.roll(a, shifts[0], dims[0])
+        first_dim_rolled = torch.roll(a, (shifts[0],), dims[0])
```

Comment:
Failed a test otherwise IIRC.

Reply:
I am fine to take this as is but it shouldn't fail 🧐